source/application/PGM/PGMAppController.cpp

```diff
@@ -104,6 +106,9 @@ void PGMAppController::createGeneralPanes()
 	PGMSlitControlView *slitView = new PGMSlitControlView();
     QWidget *slitWidget = mw_->buildMainWindowPane("Slits", ":/utilities-system-monitor.png", slitView);
     mw_->addPane(slitWidget, "General", "Slits", ":/utilities-system-monitor.png");
+
+	mw_->addPane(mw_->buildMainWindowPane("Mono Grating", ":/utilities-system-monitor.png", new PGMGratingView), "General", "Mono Grating", ":/utilities-system-monitor.png");
+	mw_->addPane(mw_->buildMainWindowPane("Undulator", ":/utilities-system-monitor.png", new PGMUndulatorView), "General", "Undulator", ":/utilities-system-monitor.png");
```

Comment:
we have attributes for the magic strings: `generalPaneCategoryName_` and `generalPaneIcon_`